src/api/route-services/projects.js

```diff
@@ -61,13 +75,8 @@ class ProjectsService {
 
     const dynamodb = createDynamoDbInstance();
 
-    try {
-      await dynamodb.updateItem(params).send();
-      return OK();
-    } catch (e) {
-      if (e.statusCode === 400) throw new NotFoundError('Project not found');
```

Comment:
Removed the try catch because I think a 400 could mean many things, not necessarily that the project was not found, so this custom error might cause more confusion that be useful

Reply:
Did you consider leaving the try/catch, but removing the if statement, so that it catches, logs and returns any dynamoDB error that can occur?